recipes/mikado/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: d77bf5ca843d8cea35208c52750d014d704d46889633d5ac0f12edba6b5aa131
 
 build:
-  number: 0
+  number: 1
```

Comment:
I don't think you need to bump the build number for changes in the `extra:` section, unless you want to force a rebuild.

Reply:
Hi @nsoranzo 

I was not aware of that, I increased it because in the past I had encountered problems by not doing so (ie the linter saying that a build of that number already existed). Should I try to revert the build number to 0?

Reply:
Ciao Luca,
not a big issue, but yes, you can amend the commit removing this change. Also, you can add `[lint skip build_number_needs_bump for recipes/mikado]` to the commit message to skip the lint check, see https://bioconda.github.io/contributor/linting.html#skipping-on-a-per-commit-basis

Reply:
Done!

Reply:
Thanks! I see you've added a new import test in the mean time, so I suppose that's worth the build number bump to actually test it on the CI. Sorry we are going back and forth on this detail :bow: